.github/workflows/ci_linux.yml

```diff
@@ -11,17 +11,17 @@ on:
         type: string
       expect_failure:
         type: boolean
-      linux_use_prebuilt_artifacts:
+      use_prebuilt_artifacts:
         type: string
```

Comment:
I figured having `linux_` in the input name was redundant in `ci_linux.yml`. Could revert that change if you want to keep it @geomin12